index.js

```diff
@@ -105,7 +153,19 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  let filtered = [];
+
+for(let i = 0; i < movies.length; i++){
+  lcGenre = genre.toLowerCase()
```

Comment:
I'd call this `downcasedGenre`. When I first saw this is said to myself "what on earth is an lcGenre?"